### DISCUSSION TITLE
Averaging peaks

### DISCUSSION THREAD

**Discussion by nesetozel** [stuart-lab/signac#722] [Q&A] [answered]
Hello @timoast 

What would be the best way to average scATAC-seq counts to generate a "pseudobulk" count matrix?
- Average the TF-IDF normalized values? I am not sure if averaging already normalized data is valid though, like, Seurat by default doesn't do it for averaging RNA...
- Average raw counts and re-run TF-IDF? I was hesitant about this because TF-IDF sounds like something intended for very sparse data.
- Average raw counts and do log-norm as if RNA?
Thank you very much!
Best,
Neset

**Comment by timoast** [comment, marked as answer]
You should use the `AverageExpression()`  function with `slot="counts"` to average the counts in each group of cells.

**Reply by nesetozel** [reply]
Thank you!
